tests/tpch/conftest.py

```diff
@@ -142,7 +141,6 @@ def tpch_database_table_schema(request, testrun_uid, scale, query, local):
         originalname=request.node.originalname,
         path=str(request.node.path.relative_to(TEST_DIR)),
         dask_version=dask.__version__,
-        dask_expr_version=dask_expr.__version__,
```

Comment:
this is a DB change. I'm not sure if any of our CI jobs reacts to this. Technically, this would require a DB migration (but I'm also fine with just setting this to null)

Reply:
If I understand correctly, this would just set the `TPCHRun.dask_expr_version` to `None`, which is allowed: https://github.com/coiled/benchmarks/blob/615ba99c901745b58042f287ef73ddd2dc169b00/benchmark_schema.py#L86. To change the actual data model, we do indeed need a migration and should remove the linked line from the ORM model.

Reply:
> I'm not sure if any of our CI jobs reacts to this.

If you're concerned with CI jobs accessing the attribute, this PR should remove all of those instances.

Reply:
ok, that's what I wanted. Thanks for checking